prereise/gather/demanddata/bldg_electrification/floor_area_avg_slopes.py

```diff
@@ -574,20 +921,237 @@ def main_plots(iso, zone_shape, pumas_shp, state_shp, plot_show=True):
+            "NYISO-B",
+            "NYISO-C",
+            "NYISO-D",
+            "NYISO-E",
+            "NYISO-F",
+            "NYISO-G",
+            "NYISO-H",
+            "NYISO-I",
+            "NYISO-J",
+            "NYISO-K",
+            "ERCO-C",
+            "ERCO-E",
+            "ERCO-FW",
+            "ERCO-N",
+            "ERCO-NC",
+            "ERCO-S",
+            "ERCO-SC",
+            "ERCO-W",
+            "CISO-PGAE",
+            "CISO-SCE",
+            "CISO-SDGE",
+            "IID",
+            "WALC",
+            "LADWP",
+            "TID",
+            "BANC",
+            "ISONE-Massachusetts",
+            "ISONE-Maine",
+            "ISONE-New Hampshire",
+            "ISONE-Vermont",
+            "ISONE-Connecticut",
+            "ISONE-Rhode Island",
+            "PJM_AE",
+            "PJM_AEP",
+            "PJM_AP",
+            "PJM_ATSI",
+            "PJM_BGE",
+            "PJM_ComEd",
+            "PJM_DAY",
+            "PJM_DEO&K",
+            "PJM_DLCO",
+            "PJM_DP&L",
+            "PJM_Dominion",
+            "PJM_EKPC",
+            "PJM_JCP&L",
+            "PJM_METED",
+            "PJM_PECO",
+            "PJM_PENELEC",
+            "PJM_PEPCO",
+            "PJM_PPL",
+            "PJM_PSEG",
+            "PJM_RECO",
+            "SPP-CSWS",
+            "SPP-EDE",
+            "SPP-GRDA",
+            "SPP-KACY",
+            "SPP-KCPL",
+            "SPP-LES",
+            "SPP-MPS",
+            "SPP-NPPD",
+            "SPP-OKGE",
+            "SPP-OPPD",
+            "SPP-SECI",
+            "SPP-SPRM",
+            "SPP-SPS",
+            "SPP-WAUE",
+            "SPP-WFEC",
+            "SPP-WR",
+            "MISO-0001",
+            "MISO-0004",
+            "MISO-0006",
+            "MISO-0027",
+            "MISO-0035",
+            "MISO-8910",
+            "PNM",
+            "EPE",
+            "TEPC",
+            "Arizona",
+            "WACM",
+            "PACE",
+            "PSCO",
+            "PSEI",
+            "DOPD",
+            "AVA",
+            "CHPD",
+            "GCPD",
+            "BPAT",
+            "PGE",
+            "PACW",
+            "IPCO",
+            "MT_west",
+            "NEVP",
+            "AECI",
+            "SOCO",
+            "AEC",
+            "TVA",
+            "Carolina",
+            "Florida",
+        ],
+        "Outliers": [
+            "NYISO-D",
+            "NYISO-H",
+            "WALC",
+            "PJM_RECO",
+            "SPP-WFEC",
+            "PSEI",
+            "AECI",
+        ],
     }
 
     iso_name = {
         "NY": "New York",
         "TX": "Texas",
         "CA": "California",
         "NE": "New England",
+        "PJM": "PJM Interconnection",
+        "SPP": "Southwest Power Pool",
+        "MISO": "Midcontinent ISO",
+        "SW": "Southwest",
+        "NW": "Northwest",
+        "SE": "Southeast",
+        "USA": "United States",
+        "Outliers": "Model Outliers",
```

Comment:
Again, if we define all these constant lists, `zone_names`, `zone_name_shps`, `iso_name`,  in `const.py`, they could be used directly or after slightly modified elsewhere and increate readability as well.

Reply:
Made the changes